Adyen/Model/Checkout/UpdatePaymentLinkRequest.cs

```diff
@@ -0,0 +1,155 @@
+#region Licence
+
+// 
+//                        ######
+//                        ######
+//  ############    ####( ######  #####. ######  ############   ############
+//  #############  #####( ######  #####. ######  #############  #############
+//         ######  #####( ######  #####. ######  #####  ######  #####  ######
+//  ###### ######  #####( ######  #####. ######  #####  #####   #####  ######
+//  ###### ######  #####( ######  #####. ######  #####          #####  ######
+//  #############  #############  #############  #############  #####  ######
+//   ############   ############  #############   ############  #####  ######
+//                                       ######
+//                                #############
+//                                ############
+// 
+//  Adyen Dotnet API Library
+// 
+//  Copyright (c) 2020 Adyen B.V.
+//  This file is open source and available under the MIT license.
+//  See the LICENSE file for more info.
+
+#endregion
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.IO;
+using System.Runtime.Serialization;
+using System.Text;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+
+namespace Adyen.Model.Checkout
+{
+    /// <summary>
+    /// UpdatePaymentLinkRequest
+    /// </summary>
+    [DataContract]
+    public partial class UpdatePaymentLinkRequest : IEquatable<UpdatePaymentLinkRequest>, IValidatableObject
```

Comment:
I think this is for the Patch call: https://docs.adyen.com/api-explorer/#/CheckoutService/v66/patch/paymentLinks/{linkId}
We don't support PATCH methods yet, right?

Reply:
There is an internal ticket for this. We can keep the class